articles/azure/azs-how-create-vm-ps.md

```diff
@@ -0,0 +1,288 @@
+>   <option value="/SQL2016SP1-WS2016/Skus/Standard">SQL Server 2016 SP1 Standard on Windows Server 2016</option>
+>   <option value="/SQL2016SP2-WS2016/Skus/Enterprise">SQL Server 2016 SP2 Enterprise on Windows Server 2016</option>
+>   <option value="/SQL2016SP2-WS2016/Skus/Standard">SQL Server 2016 SP2 Standard on Windows Server 2016</option>
+>   <option value="/SQL2017-SLES12SP2/Skus/Enterprise">SQL Server 2017 Enterprise on SLES 12 SP2</option>
+>   <option value="/SQL2017-WS2016/Skus/Enterprise">SQL Server 2017 Enterprise Windows Server 2016</option>
+>   <option value="/SQL2017-SLES12SP2/Skus/Standard">SQL Server 2017 Standard on SLES 12 SP2</option>
+>   <option value="/SQL2017-WS2016/Skus/Standard">SQL Server 2017 Standard on Windows Server 2016</option>
+>   <option value="/WindowsServer/Skus/2012-Datacenter">Windows Server 2012 Datacenter - Pay as you use</option>
+>   <option value="/WindowsServer/Skus/2016-Datacenter">Windows Server 2016 Datacenter - Pay-as-you-use</option>
+>   <option value="/WindowsServer/Skus/2016-Datacenter-Server-Core">Windows Server 2016 Datacenter - Server Core - Pay as you use</option>
+>   <option value="/WindowsServer/Skus/2016-Datacenter-with-Containers">Windows Server 2016 Datacenter - with Containers - Pay as you use</option>
+>   <option value="/CentOS/Skus/6.10">CentOS-based 6.10</option>
+>   <option value="/CentOS/Skus/6.9">CentOS-based 6.9</option>
+>   <option value="/CentOS/Skus/7.3">CentOS-based 7.3</option>
+>   <option value="/CentOS/Skus/7.5">CentOS-based 7.5</option>
+> </select></form>
+
+From your PowerShell window:
+
+<pre><code class="language-PowerShell"># Define a credential object to store the username and password for the virtual machine
+$UserName='<output form="vmusername" name="result" style="display: inline;">&lt;VM Username&gt;</output>'
+$Password='<output form="vmpassword" name="result" style="display: inline;">&lt;VM Password&gt;</output>'| ConvertTo-SecureString -Force -AsPlainText
+$Credential=New-Object PSCredential($UserName,$Password)
+
+# Create the virtual machine configuration object
+$VirtualMachine = New-AzureRmVMConfig -VMName '<output form="vmname" name="result" style="display: inline;">&lt;VM Name&gt;</output>' -VMSize '<output form="vmsize" name="result" style="display: inline;">Basic_A0</output>'
+
+# Set the VM Size and Type
+$VirtualMachine = Set-AzureRmVMOperatingSystem -VM $VirtualMachine <output form="vmtype" name="result" style="display: inline;">-Linux</output> -ComputerName '<output form="compname" name="result" style="display: inline;">&lt;Computer Name&gt;</output>' -Credential $Credential
+
+# Get the VM Source Image
+$Image = Get-AzureRMVMImagePublisher -Location 'frn00006' | Get-AzureRmVMImageOffer | Get-AzureRmVMImageSku | where {$_.Id -like '*<output form="vmimage" name="result" style="display: inline;">/UbuntuServer/Skus/14.04.5-LTS</output>*'}

```

Comment:
Something funky going on with italics for the rest of the article. Doesn't seem to affect the preview though, so maybe not an issue.

Reply:
Think its the underscore, but escaping it may mess with the code. Don't think it should actually affect the article though as it's within a code block.